packages/plugin-fontawesome/src/FontAwesomeDomObjectRenderer.ts

```diff
@@ -20,11 +20,28 @@ export class FontAwesomeDomObjectRenderer extends NodeRenderer<DomObject> {
         node: FontAwesomeNode,
         worker: RenderingEngineWorker<DomObject>,
     ): Promise<DomObject> {
+        const select = (ev: Event) => {
+            ev.preventDefault();
+            const selectFontAwesome = () => {
+                this.engine.editor.selection.select(node, node);
+            };
+            // The normaliser wait a tick before changing the selection when the user click
+            // in order to override the normalizer default behavior we also need to wait a tick
+            setTimeout(() => {
+                this.engine.editor.execCommand(selectFontAwesome);
+            });
+        };
         const fontawesome: DomObjectElement = {
             tag: node.htmlTag,
             attributes: {
                 class: new Set(node.faClasses),
             },
+            attach: (el: HTMLElement): void => {
+                el.addEventListener('mouseup', select, true);
+            },
+            detach: (el: HTMLElement): void => {
+                el.removeEventListener('mouseup', select, true);
+            },
```

Comment:
Don't we only want that in Odoo?

Reply:
I don't know, I based this on similar behavior for images, that I found in JW.
@Zinston made changes on this in images recently, and suggested to do the same for FontAwesome

Reply:
Maybe it's what we need. cc @dmo-odoo @Zinston 
I believe it belongs to click rather than `mouseup`. But for that, the `Normalizer` need the detection mechanism mention above.

Reply:
@Zinston expecitely changed it from clic to mouseUp in his PR. I guess he has a good reason.

Reply:
cf : https://github.com/odoo-dev/jabberwock/pull/325

Reply:
@Goaman I just merged @Zinston's PR so if this click/mouseup shenanigans need to be changed with the concurrency changes, I suggest you to rebase your branch on master to check if it works or if it can/needs to be changed.

Reply:
Then you can report to @sebgeelen with your findings.